lib/builderator/config/defaults.rb

```diff
@@ -0,0 +1,100 @@
+require_relative './file'
+
+module Builderator
+  # :nodoc
+  module Config
+    ##
+    # Global predefined defaults
+    ##
+    GLOBAL_DEFAULTS = File.new({}, :source => 'GLOBAL_DEFAULTS') do
+      cleanup true
+      version '0.0.0'
+      build_number 0
+
+      autoversion do |autoversion|
+        autoversion.create_tags true
+        autoversion.search_tags true
+      end
+
+      local do |local|
+        local.vendor_path 'vendor'
+        local.cookbook_path 'cookbooks'
+
+        local.data_bag_path 'vendor/chef/data_bags'
+        local.environment_path 'vendor/chef/environments'
```

Comment:
Not everyone has access to the repo with these directories.  Can we possibly render this like so and fix the cookbooks accordingly:

    chef.data_bags_path = './vendor/chef/data_bags' if ::File.exist?(::File.expand_path('../vendor/chef/data_bags', __FILE__))
    chef.environments_path = './vendor/chef/environments' if ::File.exist?(::File.expand_path('../vendor/chef/environments', __FILE__))